### PR TITLE
Create FLI page with components and market data

### PR DESCRIPTION
**Pull request by evelyncc**
### Changes 
* Create FLI page with components and market data
* New method to fetch portfolio data to populate symbol, logo url, name, and index components

### To do
* Uniswap widget
* Add FLI to nav bar
* Update description
 
### Web
image

### Mobile
image

### CODE REVIEW

File: src/contexts/Balances/Provider.tsx [hunk 1]
```diff
@@ -80,6 +83,9 @@ const Provider: React.FC = ({ children }) => {
       setDpiBalance(
         new BigNumber(balances[2]).dividedBy(new BigNumber(10).pow(18))
       )
+      setFliBalance(
+        new BigNumber(balances[2]).dividedBy(new BigNumber(10).pow(18))
```
Comment (controtie) [inline src/contexts/Balances/Provider.tsx:87]: need to increment the array index here and index for all balances afterwards (below)

File: .env.prod [hunk 2]
```diff
@@ -3,7 +3,9 @@ REACT_APP_ENVIRONMENT=staging
 REACT_APP_ETHEREUM_RPC_URL=https://mainnet.eth.aragon.network/
 REACT_APP_SNAPSHOT_URL=https://hub.snapshot.page/api
 REACT_APP_TOKENSETS_API=https://api.tokensets.com/public
```
Comment (controtie) [inline .env.prod:5]: i guess we can deprecate this env in favor of `REACT_APP_TOKENSETS_URL`?

File: src/utils/tokensetsApi.ts [hunk 3]
```diff
@@ -32,18 +32,31 @@ export const fetchSetComponents = (set: string) => {
...
-            ...comp,
-            [camelCase(k)]: component[k],
-          }),
-          {}
-        )
-        return camelCasedComponent
-      })
 
-      return formattedComponents
+      return formatComponents(components)
```
Comment (controtie) [inline src/utils/tokensetsApi.ts:36]: 👍

**Review by controtie — CHANGES_REQUESTED** [review]
LGTM. Minor comments + 1 change on set balances.
And thanks for cleaning up some of our tech debt! 🥳

File: src/contexts/Balances/Provider.tsx (hunk 1 above)
Comment (evelyncc) [reply src/contexts/Balances/Provider.tsx]: good catch!

File: .env.prod (hunk 2 above)
Comment (evelyncc) [reply .env.prod:5]: that's the plan, unfortunately the `/public` makes it hard to reuse

**Review by controtie — APPROVED** [review]
Looks great, I'll handle the last bits like updating the set description once it's ready.

**Pull request merged**